roles/pulp/README.md

```diff
@@ -17,7 +17,8 @@ Role Variables:
   Defaults to "{}", which will not install any plugins.
   * Dictionary Key: The pip installable plugin name. This is defined in each
   plugin's* `setup.py`. **Required**.
-  * `upgrade`: Whether to update/upgrade the plugin to the latest stable release from PyPI. Only affects systems where the plugin is already installed. If `source_dir` is set, this has no effect and is effectively always `true`. Note that if the latest stable release of the plugin is incompatible with pulpcore's version, ansible-pulp will fail (and exit the play) when it goes to upgrade the plugin. Defaults to "false".
+  * `version`: Specific release of the plugin to install from PyPI, or upgrade to (regardless of whether `upgrade` is set.) If `source_dir` is set, this has no effect. Note that if the spceified release of the plugin is incompatible with pulpcore's version, ansible-pulp will fail (and exit the play) when it goes to install or upgrade the plugin. Defaults to nothing.
```

Comment:
@mikedep333 to confirm, this and the variable `upgrade` below are only for upgrading plugins right?

Reply:
Correct.